### ISSUE TITLE
Breaking change in timeInterval

### ISSUE THREAD

**Issue by jlblcc** [adiwg/mdTranslator#199]
The `interval` property in the mdJSON reader timeInterval module throws an error if the interval is not an integer. This is a breaking change! The current mdJSON schema example fails this test, which also means the mdJSON example is broken for downstream applications(mdTools). So we should probably issue a warning for this until 3.0.

See here:
https://github.com/adiwg/mdTranslator/blob/c99791d8a9e7028650a44b0c87235ce5b8cf9618/lib/adiwg/mdtranslator/readers/mdJson/modules/module_timeInterval.rb#L33

**Comment by jlblcc** [comment]
I monkeypatched mdTools example until this is resolved.

**Comment by stansmith907** [comment]
This is true. The ISO standard calls for integers. I double checked the documentation before adding these tests to the latest release. It also makes sense to require these fields as integers except possible for seconds. We should decide if we want mdJson act differently.
